### PR TITLE
Add missing cmake in various distro packages

### PR DESCRIPTION
**Pull request by vepadulano**

### CODE REVIEW

**Comment by Axel-Naumann** [comment]
It's that still needed? Some distro cmake packages were too old and I accidentally removed the download line from the Dockerfile of these images - that's fixed now. Not sure about alma9 though...

File: ubuntu20/packages.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 automake
 binutils
 ca-certificates
+cmake
```
Comment (Axel-Naumann) [inline ubuntu20/packages.txt:4]: Ubuntu 20's cmake is too old, we are instead downloading a release in its `Dockerfile`:
```suggestion
```

**Review by Axel-Naumann — APPROVED** [review]
Apart from ubuntu20: lgtm! Almas will install cmake 3.20 through their AppStream repo.

**Comment by vepadulano** [comment]
Thanks @Axel-Naumann ! The failure on Alma 9 doesn't seem related to me, should we merge?

**Pull request merged**